hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/flink/FlinkCleaner.java

```diff
@@ -67,7 +67,7 @@ public void deleteOrphanJobs(Timer timer) {
         }
       }
     } catch (Exception ex) {
-      LOGGER.log(Level.SEVERE, "Could not access " + settings.getFlinkConfFile(), ex);
+      LOGGER.log(Level.SEVERE, "Could not access flink configuration file", ex);
```

Comment:
@robzor92 why we don't want to print the file location?

Reply:
I wanted to avoid it as we potentially hit the database, maybe not with the current logic the risk is not too high. Basically I did not want us getting an exception in the statement where we are catching Exception to avoid timer expungeon. This was before I enabled rescheduling of failed timers so I suppose I could put it back.

Reply:
Never mind, not that important